### PR TITLE
Update permissions.yml

### PR DESCRIPTION
**Pull request by MrCraft888**

### CODE REVIEW

**Comment by mythguy1226** [comment]
remove iceman's name at the top and you copied abominations permissions so now there is 2 remove the one you copied

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+        - ch.alias.haste0
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5  
+        - ch.alias.haste6
+        - ch.alias.haste7
+        - ch.alias.haste8
+   iceman:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: too many spaces on this line

Comment (MrCraft888) [inline PermissionsEx/permissions.yml]: ok

**Comment by MrCraft888** [comment]
there

**Pull request merged**